.github/workflows/ci.yml

```diff
@@ -8,14 +8,16 @@ on:
 
 jobs:
   test:
+    name: Test
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         node-version: [20, 22, 24]
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Hey @kibertoad,
I see you removed windows from CI in #184, 
the run was failing due to an issue with linting. 
I don't think that lint task should be executed in every node / os combination so I created a separate task.